specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/CopyOperationResult.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "resultId": "{resultId}",
+    "body": {}
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "status": "succeeded",
+        "createdDateTime": "2020-01-01T00:00:00Z",
+        "lastUpdatedDateTime": "2020-01-01T00:01:00Z",
+        "copyResult": {
+          "modelId": "{modelId}",
```

Comment:
Will the model get a new Id in the subscription it's copied to, or will the Id be the same on both subscriptions?

Reply:
The model will get a new Id.